tutorials/heritage/index.md

```diff
@@ -0,0 +1,6 @@
+\defgroup tutorial_heritage Heritage tutorials
+\ingroup Tutorials
+\brief Tutorials not anymore in line with times
+
+ROOT is a rapidly evolving toolkit, as well as many of the libraries and languages it relies on.
+Some of the examples which were written during its history had an important role once exposed to users but today they are not anymore in line with times.
```

Comment:
Possibly clearer (what is "in line with times"?):
```
"Some of ROOT's tutorials demonstrate interfaces that are not recommended anymore, because ROOT or C++ itself now offer superior ones. These have been moved here "for the record".
```

Reply:
OK.